e2e/tests/functional/recentObjects.e2e.spec.js

```diff
@@ -252,6 +252,30 @@ test.describe('Recent Objects', () => {
         // Assert that the list is empty
         expect(await recentObjectsList.locator('.c-recentobjects-listitem').count()).toBe(0);
     });
+    test("Clears the recent objects and disables the button", async ({ page }) => {
+        test.slow();
+
+        // Assert that the list initially contains 3 objects (clock, folder, my items)
+        expect(await recentObjectsList.locator('.c-recentobjects-listitem').count()).toBe(3);
+
+        // Click the aria-label="Clear Recently Viewed" button
+        await page.getByRole("button", { name: "Clear Recently Viewed" }).click();
+
+        // Click on the "OK" button in the confirmation dialog
+        await page.getByRole("button", { name: "OK" }).click();
+
+        // Assert that the list is empty
+        expect(
+            await recentObjectsList.locator(".c-recentobjects-listitem").count()
+        ).toBe(0);
+
+        // Assert that the button is disabled
+        expect(
+            await page
+                .getByRole("button", { name: "Clear Recently Viewed" })
+                .isEnabled()
+        ).toBe(false);
```

Comment:
Awesome. Let's also have the test navigate to one object and verify that the button is enabled again.

Reply:
Add test.

Reply:
Let's just add these checks in the original test you added so as not to duplicate actions. e2e tests are costly to run, so longer running tests with more assertions are preferred.